shoes-core/lib/shoes/common/inspect.rb

```diff
@@ -9,7 +9,7 @@ def to_s
       # Example:
       #   (Shoes::App:0x01234abc "Hello")
       def inspect
-        "(#{self.class.name}:#{'0x%08x' % (object_id * 2)}#{inspect_details})"
+        sprintf('(%s:%s%s)', self.class.name, hexy_object_id, inspect_details)
```

Comment:
there is a rubocop offense here? That wants you to use `sprintf` ? I'd need to read the style guid to probably get the reasoning but I never used `sprintf` before in my ruby life so I'd be against adding it and eventually turning off the cop.